.devcontainer/devcontainer.json

```diff
@@ -9,7 +9,7 @@
 		"python.linting.pylintEnabled": true,
 		"python.linting.pylintPath": "/usr/local/bin/pylint",
 		"python.pythonPath": "/usr/local/bin/python",
-		"terminal.integrated.fontFamily": "FiraCode Nerd Font Mono, MonoLisa, monospace"
+		"terminal.integrated.fontFamily": "Fira Mono Medium Nerd Font Complete Mono, MonoLisa, monospace"
```

Comment:
Feel free to get rid of starship utility and FiraCode font if that gives too much trouble (although I do like these especially for VSCode).